.buildkite/pipelines/on_merge.yml

```diff
@@ -78,10 +78,19 @@ steps:
 
   - command: .buildkite/scripts/steps/checks.sh
     label: 'Checks'
+    agents:
+      queue: n2-2-spot
+    timeout_in_minutes: 60
+    retry:
+      automatic:
+        - exit_status: '-1'
+          limit: 3
+
+  - command: .buildkite/scripts/steps/check_types.sh
+    label: 'Check Types'
     agents:
       queue: c2-8
```

Comment:
I'm pretty sure that Check Types is single threaded and doesn't need all 8 cores, still probably good to go with c2 but maybe c2-4?

Reply:
n2-4 actually timed out, and the metrics show a CPU load of about 8, so it seems to be using all of the cores